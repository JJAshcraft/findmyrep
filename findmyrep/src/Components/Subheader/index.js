import React from "react";
import Grid from "@material-ui/core/Grid";
import styled from "styled-components";

const StyledGrid = styled(Grid)`
  padding: 10px 0 10px 0;
`;

const Subheader = props => {
  return (
    <StyledGrid container spacing={32}>
      <Grid item xs={12}>
        {props.children}
      </Grid>
    </StyledGrid>
  );
};

export default Subheader;
