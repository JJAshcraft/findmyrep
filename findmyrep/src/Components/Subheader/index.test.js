import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Subheader from "./index.js";

test("HR component renders with proper styling", () => {
  const tree = renderer.create(<Subheader />).toJSON();
  expect(tree).toMatchSnapshot();
});
