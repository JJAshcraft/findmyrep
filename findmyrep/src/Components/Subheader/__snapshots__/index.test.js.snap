// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HR component renders with proper styling 1`] = `
.c0 {
  padding: 10px 0 10px 0;
}

<div
  className="MuiGrid-container-1 MuiGrid-spacing-xs-32-26 c0"
>
  <div
    className="MuiGrid-item-2 MuiGrid-grid-xs-12-41"
  />
</div>
`;
